geth/jail/internal/timers/timers.go

```diff
@@ -9,62 +9,81 @@ import (
 	"github.com/status-im/status-go/geth/jail/internal/vm"
 )
 
-var minDelay = map[bool]int64{
-	true:  10,
-	false: 4,
-}
+type intervalFunc func(*loop.Loop) func(call otto.FunctionCall) otto.Value
 
-//Define jail timers
-func Define(vm *vm.VM, l *loop.Loop) error {
+// Define jail timers
+func Define(vm *vm.VM, l *loop.Loop) (err error) {
```

Comment:
As long as I try to use named return values more, I am not sure it's a good example because when getting `setTimeout` it's returned explicitly and later implicitly while using the same `if err := ...; err != nil` pattern. What do you think about returning `error` always explicitly in this function?

Reply:
Yep, agree it could be confusing on this function, i'll change that.